Source/WebCore/html/HTMLImageElement.cpp

```diff
@@ -501,6 +509,15 @@ void HTMLImageElement::didAttachRenderers()
         renderImage->setImageSizeForAltText();
 }
 
+void HTMLImageElement::didAddUserAgentShadowRoot(ShadowRoot& shadowRoot)
```

Comment:
I don't think we want to override this function.
Instead, ensureSpatialControls should call `ensureUserAgentShadowRoot()` and add the spatial control.

Reply:
Removed the override of didAddUserAgentShadowRoot